library/globals.inc.php

```diff
@@ -1772,13 +1772,28 @@ function gblTimeZones()
     //
     'Notifications' => array(
 
+
         'patient_reminder_sender_name' => array(
             xl('Patient Reminder Sender Name'),
             'text',                           // data type
             '',                               // default
             xl('Name of the sender for patient reminders.')
         ),
 
+      'SMS_GATEWAY_RETURN_NUMBER' => array(
+          xl('SMS Gateway Return Number'),
+          'text',                           // data type
+          '',                               // default
+          xl('Require within the United States.  Must sign up for two-way messaging.')
```

Comment:
this is weird, but in the english constants that get translated. don't have more than 1 consecutive space since this can cause problems in the translation pipeline.
xl('Require within the United States. Must sign up for two-way messaging.')